pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/ReplicatedSubscriptionsController.java

```diff
@@ -206,6 +206,12 @@ private void receiveSubscriptionUpdated(ReplicatedSubscriptionsUpdate update) {
     private void startNewSnapshot() {
```

Comment:
I think the better way is not to create `ReplicatedSubscriptionsController` when turning off `replication.`

Reply:
I am trying to use this way for the first time.
But If we do not create `ReplicatedSubscriptionsController ` when turning off `replication`, the `ReplicatedSubscriptionsController` cannot be created when enabling replication at the namespace level.
```
bin/pulsar-admin namespaces set-clusters my-tenant/my-namespace \
--clusters us-west,us-east,us-cent
``` 

Do you have any idea you do this? Adding a new API to start `ReplicatedSubscriptionsController` for all topics in the namespace in all the brokers maybe can do that, but that adds too much complexity.